### PR TITLE
Various additions for 0.5.0

### LINKED ISSUE #40: Add datapacks
See https://git.minetest.land/VoxeLibre/VoxeLibre/pulls/4520

### LINKED ISSUE #94: Add new banner patterns, and pottery sherds from 1.21
@crispiebacon We have media, but ofc. Once again I'll need you to make new textures.

### LINKED ISSUE #112: Add dripstone blocks

### LINKED ISSUE #113: New bundles textures
@crispiebacon Since the bundles are in game already and their textures only differ 22% from MC which cuts it very very close. I need these new textures kind of soon. The lowest it could possibly be 20%

### PR DESCRIPTION
**Pull request by DragonWrangler1**
closes #112 
closes #113 
closes #94 
closes #40

### CODE REVIEW

File: mods/HELP/vlf_doc_basics/README.md
Comment (crispiebacon) [inline mods/HELP/vlf_doc_basics/README.md:1]: also wrong naming.

File: mods/HELP/vlf_doc_basics/basics.lua
Comment (crispiebacon) [inline mods/HELP/vlf_doc_basics/basics.lua:1]: entity effect issue again.

File: mods/HUD/vlf_inventory/API.md
Comment (crispiebacon) [inline mods/HUD/vlf_inventory/API.md:1]: vlf ?

File: mods/HUD/vlf_inventory/README.txt
Comment (crispiebacon) [inline mods/HUD/vlf_inventory/README.txt:1]: same as above.

**Review by crispiebacon — CHANGES_REQUESTED** [review]
I'll leave it at 24 cases for now. 1k files is a lot to review.

**Comment by DragonWrangler1** [comment]
This clearly needs more testing.

**Pull request closed without merging**